### PR TITLE
Merge latest commits from Sufia

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: lib/generators/hyrax/templates/catalog_controller.rb [hunk 1]
```diff
@@ -50,6 +50,10 @@ def self.modified_field
     # It's used to give a label to the filter that comes from the user profile
     config.add_facet_field solr_name("generic_type", :facetable), label: "Type", if: false
 
+    # The generic_type isn't displayed on the facet list
+    # It's used to give a label to the filter that comes from the user profile
+    config.add_facet_field solr_name("generic_type", :facetable), label: "Type", if: false
```
Comment (jcoyne) [inline lib/generators/hyrax/templates/catalog_controller.rb]: This is s duplicate of line 51, please remove.

Comment (mjgiarlo) [inline lib/generators/hyrax/templates/catalog_controller.rb]: :+1:

**Comment by mjgiarlo** [comment]
@jcoyne Done.

**Comment by jcoyne** [comment]
@mjgiarlo can you squash the last 3 commits together?

**Comment by mjgiarlo** [comment]
@jcoyne I can't find a way to do that cleanly given that I've got a merge commit in the mix.

**Comment by jcoyne** [comment]
@mjgiarlo I think those changes are part of the merge, thus they should all be one commit.

**Comment by mjgiarlo** [comment]
@jcoyne I don't disagree, but I've tried to squash those two or three different ways now and none of them do what I would expect.

**Comment by mjgiarlo** [comment]
@jcoyne figured it out. Long story short: the `-p` flag to `git-rebase` is lovely.

**Pull request merged**